### PR TITLE
Paragraph styles

### PR DESCRIPTION
**Pull request by MalcolmVonMoJ**
Minor changes highlighted during the paragraph work in MoJ Blocks.  
This should be deployed alongside [PR156](https://github.com/ministryofjustice/wp-moj-blocks/pull/156) in MoJ Blocks repo.

### CODE REVIEW

File: assets/scss/moj-blocks-general.scss [hunk 1]
```diff
@@ -7,25 +7,10 @@
 }
 
 .govuk-main-wrapper {
-  p:not([class]), p[class=""] {
```
Comment (RobLoweMOJ) [inline assets/scss/moj-blocks-general.scss:10]: Did we check removing this didnt cause any list style issues?

Comment (MalcolmVonMoJ) [reply assets/scss/moj-blocks-general.scss]: We did check and we decided that these styles should all be left in - so I have done that.  

This Hale element of the change is now far less important, but there are a few things.

**Review by RobLoweMOJ — DISMISSED** [review]
LGTM

**Review by RobLoweMOJ — APPROVED** [review]
LGTM

**Pull request merged**